workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/hooks/useCompanyOptions.ts

```diff
@@ -36,6 +39,11 @@ export default function useCompanyOptions(
 			`/o/${LiferayAPIs.HEADERLESS_ADMIN_USER}/accounts/by-external-reference-code/${selectedAccountBrief.externalReferenceCode}`
 	);
 
+	const {data: currencies} = useGet<Currencies>(
+		account &&
+			`/o/headless-commerce-admin-catalog/v1.0/currencies?filter=code eq '${account.currency}'`
```

Comment:
The hope is to pull this from Salesforce instead of Liferay, the object is called DatedConversionRate

Reply:
@ryanschuhler, @abfranca is working on changing it

Reply:
Done!